bin/build-utils.js

```diff
@@ -50,7 +50,6 @@ function doBrowserify(pkgName, filepath, opts, exclude) {
     } else {
       bundler = browserify(addPath(pkgName, filepath), opts)
         .transform('es3ify')
```

Comment:
This needs a semicolon to pass linting...

Reply:
Yes, sorry, fixed.